src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -586,6 +586,15 @@ def decode(
         Returns:
             `str`: The decoded sentence.
         """
+        initial_prompt_start_id = self.convert_tokens_to_ids("<|startofprev|>")
```

Comment:
Perhaps this could be an instance variable.

Reply:
Now that we've updated this decode function with your above suggestion this line was removed. This same logic now is only used once and its on the processor. Would you still like me to make it an instance var there?

Reply:
If it's just used once then it's not really worth making it an instance variable.